docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -523,6 +523,14 @@ So when you access the `/health/ready` endpoint of your application you will hav
 
 This behavior can be disabled by setting the `quarkus.mongodb.health.enabled` property to `false` in your `application.properties`.
 
+== Metrics
+
+If you are using the `quarkus-smallrye-metrics` extension, `quarkus-mongodb` will automatically add metrics for the connection pools.
```

Comment:
"Will automatically add metrics" followed by explanation that it must be explicitly enabled is a bit confusing.